### PR TITLE
Set up dapi-management package

### PR DESCRIPTION
**Pull request by peterjurco**
#112 

Initial setup of the dapi-management package. We will need more data to be exported over time I suppose. 

Exporting:
```
export { dapis, supportedChains, api3Chains, api3Integrations };
```

### CODE REVIEW

**Review by mcoetzee — CHANGES_REQUESTED** [review]
Nice one 👍 
Is this PR partially completing https://github.com/api3dao/dapi-management/issues/112 or is it closing the issue? (I ask because I see you only linked to the issue vs the "Closes #112")

**Comment by mcoetzee** [comment]
@metobom I've given my review, but can you also give it a review 👍

**Review by metobom — APPROVED** [review]
I think exporting whole API integrations is nice. LGTM.

File: scripts/build-package.js
[the remark this answers is not in the document]
Comment (peterjurco) [reply scripts/build-package.js:1]: Agree. Originally I had more content in this script (before I realized I have to use `tsc`)

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "lib": ["ESNext"],
+    "module": "CommonJS",
+    "esModuleInterop": true,
+    "moduleResolution": "Node",
+    "strict": false,
+    "target": "ESNext",
+    "skipLibCheck": true,
+    "noFallthroughCasesInSwitch": true,
+    "exactOptionalPropertyTypes": false,
+    "noUncheckedIndexedAccess": false,
```
[the remark this answers is not in the document]
Comment (peterjurco) [reply tsconfig.json:14]: No we don't, you are right. These are just copy pasta I forgot to double check

**Comment by peterjurco** [comment]
> Is this PR partially completing https://github.com/api3dao/dapi-management/issues/112 or is it closing the issue?

I will double check this with Benjamin. Not sure if we want to have the task only for setting up the package or for fully define it

**Review by mcoetzee — APPROVED** [review]
🤘

**Pull request merged**